misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/IterableToContainEntriesSpecBase.kt

```diff
@@ -23,14 +24,25 @@ abstract class IterableToContainEntriesSpecBase(
         val anElementWhich = DescriptionIterableAssertion.AN_ELEMENT_WHICH.getDefault()
         val toBeLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
         val toBeGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+        val hasDescriptionBasic = DescriptionBasic.HAS.getDefault()
+        val nextElement = DescriptionIterableAssertion.NEXT_ELEMENT.getDefault()
+        val mismatches = DescriptionIterableAssertion.WARNING_MISMATCHES.getDefault()
```

Comment:
shouldn't be necessary as it is already defined in the super-class companion IterableToContainSpecBase

Reply:
I moved `hasDescriptionBasic` and `nextElement` up to the `IterableToContainSpecBase` companion.